build.js

```diff
@@ -85,7 +85,7 @@ const fencedCodeBlockDefinition = (name, identifiers, sourceScope, language, add
 
 	return `fenced_code_block_${name}:
   begin:
-    (^|\\G)(\\s*)(\`{3,}|~{3,})\\s*(?i:(${identifiers.join('|')})((\\s+|:|,|\\{|\\?)[^\`~]*)?$)
+    (^|\\G)(\\s*)(\`{3,}|~{3,})\\s*(?i:(${identifiers.join('|')})((\\s+|:|,|\\{|\\?)(?:(?!\`{3}|~{3}).)*)?$)
```

Comment:
Let's try to align this with the markdown spec: https://spec.commonmark.org/0.30/#fenced-code-blocks Here's the relevant section:

> If the [info string](https://spec.commonmark.org/0.30/#info-string) comes after a backtick fence, it may not contain any backtick characters. (The reason for this restriction is that otherwise some inline code would be incorrectly interpreted as the beginning of a fenced code block.)

Github doesn't do the same for fetched code blocks starting with `~`. As far as I can tell, the info string in those can contain as many `~` as you want

Reply:
I think the difficulty with Markdown is that there are so many different flavours, Commonmark is just one spec.

What would you recommend doing here? Since common mark only references backticks, we could potentially, be more or less strict.

What do you reckon to doing ~[^\`]*~ which would allow as many tildes as desired.

This would cause issues with `~~~foo~~~` but I can't find anything that supports using tildes like that.